### DISCUSSION TITLE
PSO Uniform

### DISCUSSION THREAD

**Discussion by M3rcuz1o** [bananarama92/MBS#3] [General]
Hi, is the PSO Uniform with the Exclusive Locks designed to keep the wearer like this forever or is there some way out?
For the unaware user, this might be a little too much, considering that no one can touch the locks, even.

**Comment by bananarama92** [comment]
Hey M3rcuz1o, exclusive locks are characterized by *only* being able to unlocked by other people; not the person wearing them. So while you can't touch them yourself other people can still very much do so.

**Reply by M3rcuz1o** [reply]
sorry, I didn't mean the exclusive locks. I've been a 2yr member now, and am familiar with exclusive locks.. I meant extreme? No idea, but that one locktype is untouchable by oneself and others too, which perma binds you if you don't cheat... Pretty weird! Is that intentional. Happy to provide screenshots, if necessary. Well, not happy, since it's a super scary situation.. but yeah... it's dangerous, especially for noobs.

**Reply by M3rcuz1o** [reply]
And thanks for the cool addon, we had a ton of fun already ^^

**Comment by M3rcuz1o** [comment]
hm, my bad, it was a configuration error ^^ everything works fine...

**Reply by bananarama92** [reply]
Ah, was it the high sec locks in combination with the BC "locks on you cannot be picked" setting?
